code-studio/src/js/progress.js

```diff
@@ -174,7 +177,7 @@ function loadProgress(scriptData, currentLevelId) {
     return state;
   }, {
     currentLevelId: currentLevelId,
-    display: teacherCourse ? 'list' : 'dots',
+    teacherCourse: scriptData.plc,
```

Comment:
teacherCourse isn't an intuitive name - if this is professional learning then can we call it professional learning course